containers/base.Dockerfile

```
@@ -6,7 +6,7 @@ ARG MAKEJOBS=4
 # Install pixelator dependencies in a separate stage to improve caching
 FROM registry.fedoraproject.org/fedora-minimal:40 AS runtime-base
```

Not strictly related but while we are updating we should probably bump this to 42.

```suggestion
FROM registry.fedoraproject.org/fedora-minimal:42 AS runtime-base
```